visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h

```diff
@@ -88,7 +88,7 @@ class PlanningSceneDisplay : public rviz::Display
   void queueRenderSceneGeometry();
 
   // pass the execution of this function call to a separate thread that runs in the background
-  void addBackgroundJob(const boost::function<void()> &job, const std::string &name);
+  void addBackgroundJob(const boost::function<void()> &job, const std::string &name, bool ownThread=false);
```

Comment:
Where do you want to have the underscores? As far as I understand it, only member variables should be suffixed by an underscore. Is there anything else?